README.md

```diff
@@ -206,6 +206,41 @@ A number of options are available to allow you to customise the SDK:
 
   Turns the SDK into a modal, which fades the background and puts the SDK into a contained box.
 
+  Example:
+  ```javascript
+  <script>
+      var onfido = {}
+
+      function triggerOnfido() {
+        onfido = Onfido.init({
+          useModal: true,
+          isModalOpen: true,
+          onModalRequestClose: function() {
+            // Update options with the state of the modal
+            onfido.setOptions({isModalOpen: false})
+          },
+          token: 'token',
+          onComplete: function(data) {
+            // callback for when everything is complete
+            console.log("everything is complete")
+          },
+          steps: [
```

Comment:
you can drop `steps` from here too.
I'm a bit hesitant with adding long examples to side features. My fear is the readme can grow quite large